acceptance/bundle/apps/test.toml

```diff
@@ -3,3 +3,7 @@ RecordRequests = true
 Ignore = [
     '.databricks',
 ]
+
+[[Repls]]
+Old = 'Uploaded \d+ files \(ignored \d+ directories( and \d+ files)? by .gitignore\)'
+New = 'Uploaded [NUMID] files (ignored [NUMID] directories by .gitignore)'
```

Comment:
Why Local and Cloud have different number of files?

Reply:
I don't yet have answer for this, need to dive deeper into this